src/main/java/Tokenise.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Arrays;
+/**
+ * Tokenise is to help split user input
+ * into individual strings for sanitisation
+ * and other checks
+ */
+public class Tokenise {
+    private String[] tokens = new String[] {};
+    private Boolean isKeyword = false;
+
+    public Tokenise(String userInput) {
+        tokens = userInput.split(" ");
+
+        switch (tokens[0]) {
+        case "list":
+        case "todo":
+        case "deadline":
+            //try catch for adding into
+            //Classes to check if date is
+            //added for the keyword
+        case "event":
+            isKeyword = true;
+            break;
+        case "mark":
+        case "unmark":
+            if (tokens.length == 2) {
+                try {
+                    Integer.parseInt(tokens[1]);
+                    tokens[1] = String.valueOf(Integer.parseInt(tokens[1])-1);
```

Comment:
Can save tokens[1] as a variable with a descriptive name to describe what does tokens[1] represent inorder to improve code readability.
Can add spacings between (tokens[1]), - and 1 to conform with the coding standard.